package-lock.json

```diff
@@ -4379,6 +4379,492 @@
       "requires": {
         "graceful-fs": "^4.1.11",
         "through2": "^2.0.3"
+<<<<<<< HEAD
```

Comment:
Merge issue here is causing the tests to fail, I think you can just discard the changes to this file.